Jenkinsfile

```diff
@@ -190,12 +190,12 @@ spec:
            description: 'Database to be used for running CTS. Currently only JavaDB is supported.' )
     choice(name: 'BUILD_TYPE', choices: 'CTS\nSTANDALONE-TCK', 
            description: 'Run the full EE compliance testsuite or a standalone tck' )
-    string(name: 'test_suites', defaultValue: 'concurrency connector ejb ejb30/bb ejb30/lite/appexception ejb30/lite/async ejb30/lite/basic ejb30/lite/ejbcontext ejb30/lite/enventry ejb30/lite/interceptor ejb30/lite/lookup ejb30/lite/naming ejb30/lite/nointerface  ejb30/lite/packaging ejb30/lite/singleton ejb30/lite/stateful ejb30/lite/tx ejb30/lite/view ejb30/lite/xmloverride ejb30/assembly ejb30/timer ejb30/webservice ejb30/zombie ejb30/misc ejb30/sec ejb32 el integration jacc jaspic javaee javamail jaxrs jbatch jdbc_appclient jdbc_ejb jdbc_jsp jdbc_servlet jms_appclient jms_ejb jms_jsp jms_servlet jpa_appmanaged jpa_appmanagedNoTx jpa_pmservlet jpa_puservlet jpa_stateful3 jpa_stateless3 jsf jsonb jsonp jsp jstl jta jws rmiiiop samples securityapi servlet signaturetest/javaee webservices12 webservices13 websocket xa',
+    string(name: 'test_suites', defaultValue: 'concurrency connector ejb ejb30/bb ejb30/lite/appexception ejb30/lite/async ejb30/lite/basic ejb30/lite/ejbcontext ejb30/lite/enventry ejb30/lite/interceptor ejb30/lite/lookup ejb30/lite/naming ejb30/lite/nointerface  ejb30/lite/packaging ejb30/lite/singleton ejb30/lite/stateful ejb30/lite/tx ejb30/lite/view ejb30/lite/xmloverride ejb30/assembly ejb30/timer ejb30/webservice ejb30/zombie ejb30/misc ejb30/sec ejb32 el integration jacc jaspic javaee javamail jaxrs jbatch jdbc_appclient jdbc_ejb jdbc_jsp jdbc_servlet jms_appclient jms_ejb jms_jsp jms_servlet jpa_appmanaged jpa_appmanagedNoTx jpa_pmservlet jpa_puservlet jpa_stateful3 jpa_stateless3 jsf jsonb jsonp jsp jstl jta jws samples securityapi servlet signaturetest/javaee webservices12 webservices13 websocket xa',
            description: 'Space separated list of Test suites to run') 
     string(name: 'standalone_tcks', defaultValue: 'caj concurrency connector el jacc jaspic jaxrs jaxws jms jpa jsf jsp jsonb jsonp jstl jta saaj securityapi servlet websocket', 
            description: 'Space separated list of standalone TCKs to build and run') 
     string(name: 'USER_KEYWORDS',
-           defaultValue: '',
+           defaultValue: '!rmiiop&!corba',
```

Comment:
I suggest we continue to run the optional tests where these are implemented. We can have a separate run to test the optional techs maybe not that frequently.

Reply:
bug states "Make RMIIOP tests configurable to be run or always disabled", hence we have passed by default disable keywords.
Also CORBA, Java IDL, RMI-IIOP are removed technologies - https://jakarta.ee/specifications/platform/9/platform-spec-9-SNAPSHOT.html#a2333
Based on above inputs, disable parameters are passed(have added back rmiiop testsuite in default ts list)

Reply:
If they are removed tech should we add *_optional for rmiiop

Reply:
rmi/iiop/corba/idl is a unique scenario in which officially tech is removed from the Jakarta EE Platform(like interop), but due to large usage base, tests are retained as optional/configurable/disabled.
We can still treat tests are configurable/optional, but maintain it in disabled state.

Reply:
IMO the rmiiiop tests should not be run in javaee, javaee_optional, javaee_web_profile_optional or default keywords if they are removed tech…if someone wants to run them use keyword rmiiiop thus they are retained and configurable.

Reply:
javaee_optional, javaee_web_profile_optional have been removed.